tests/eas/capacity_capping.py

```diff
@@ -109,6 +131,33 @@ def populate_params(cls):
             ).get()
 
     @classmethod
+    def _freeze_userspace(cls):
+        if "freeze_userspace" not in TEST_CONF.get("flags", []):
+            cls._log.info('not asked to freeze userspace');
+            return
```

Comment:
I don't think there's any need to add a flag to the test conf. If someone wants to run without freezing userspace, they're going to have to read the code to figure out how (unless you feel like writing a big README...). By the time they've done that, nobody saves any effort by having a configuration flag, they might as well just comment out the relevant code.